index.js

```diff
@@ -725,6 +725,198 @@ PersonaClient.prototype._removeTokenFromCache = function (id, secret, callback)
 };
 
 /**
+ * Get scope information for a user
+ * @param guid
+ * @param token
+ * @param callback
+ */
+PersonaClient.prototype.getScopesForUser = function(guid, token, callback) {
+    try {
+        _.map([guid, token], function (arg) {
+            if (!_.isString(arg)) {
+                throw "guid and token are required strings";
```

Comment:
Instead of the try/catch boilerplate you could just callback with the error message here and then immediately return.